### PR TITLE
Create Standards

### PR DESCRIPTION
**Pull request by IB-12**

### CODE REVIEW

**Comment by yzueger** [comment]
Maybe we could put the standards in a .md file for nice formatting when viewing?
Content looks good to me

**Comment by joriba** [comment]
Agree with yoric, Markdown would be nice

**Comment by mbouaissi** [comment]
Looks good to me, but maybe add more details, with how we should comment. Like saying if we want javadoc, or just one line comments?

File: Standards [hunk 1]
```diff
@@ -0,0 +1,5 @@
+1. One branch per feature
+2. Explicit comments and docs
+3. Clean formatting (KTFMT) (Camel Case)
+4. Use flows or livedata
+5. https://developer.android.com/kotlin/style-guide
```
Comment (tsogtb) [inline Standards:5]: Let's include: Avoid hard coded strings and use stringResource whenever possible.

**Review by mbouaissi — APPROVED** [review]
lgtm

**Comment by joriba** [comment]
Uhm, why is the last commit empty? @KlaKalma did you forget to `git add` something?

**Comment by KlaKalma** [comment]
Forgot to add sorry, I just renamed, should be good now.

**Pull request merged**